OmiseSDK/OmiseJsonParser.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-@objc(OMSOmiseJsonParser) public final class OmiseJsonParser: NSObject {
```

Comment:
Just pop the idea on how to fix for Swift type too.
`
typealias OmiseJsonParser = OmiseJSONParser
objc(OMSJSONParser) public final class OmiseJSONParser: NSObject
`
What do you think about this?